mevshare/builders.go

```diff
@@ -22,6 +22,9 @@ type BuilderAPI uint8
 const (
 	BuilderAPIRefundRecipient BuilderAPI = iota
 	BuilderAPIMevShareBeta1
+
+	orderflowHeader = "x-orderflow-origin"
+	flashbotsSource = "flashbots"
```

Comment:
suggestion:

rename this to `orderflowHeaderValue` and make it configurable by environment variable.